test.js

```diff
@@ -314,7 +349,7 @@ function assertFixture(t, rule, info, fixture, basename, settings) {
 
   try {
     proc.runSync(proc.parse(file), file)
-  } catch (error) {
+  } catch (/** @type any */ error) {
     if (error && error.source !== 'remark-lint') {
       throw error
     }
```

Comment:
@remcohaszing Could you also check that the `url`s of messages matches the expected url on GH? Something like this: https://github.com/remarkjs/remark-lint/blob/909d9cceea9007b8d4029c89c2209f428880914b/test.js#L323-L336, and around that place, but making sure that those match “https://github.com/remarkjs/remark-lint/tree/main/' + packageName + '...'”

Reply:
Yep! Great idea!